kolibri/content/test/test_content_app.py

```diff
@@ -66,50 +66,50 @@ def test_update_content_copy(self):
+        api.update_content_copy(channel_id=self.the_channel_id, file_object=file_1, content_copy_path=fpath_1)
+        file_3 = content.File.objects.using(self.the_channel_id).get(file_format=fm_3)
+        api.update_content_copy(channel_id=self.the_channel_id, file_object=file_3, content_copy_path=fpath_1)
         self.assertEqual(1, len(os.listdir(settings.CONTENT_COPY_DIR+'/0/9/')))
 
         # swap the content copy in file_3
         fpath_2 = self.temp_f_2.name
-        self.assertEqual(file_3.extension, '.pdf')
-        api.update_content_copy(file_3, fpath_2)
-        self.assertEqual(file_3.extension, '.mp4')
+        self.assertEqual(file_1.file_format.extension, 'pdf')
+        api.update_content_copy(channel_id=self.the_channel_id, file_object=file_1, content_copy_path=fpath_2)
+        self.assertEqual(file_1.file_format.extension, 'mp4')
 
         # because file_3 and file_2 all have reference pointing to this content copy,
         # erase the reference from file_2 won't delete the content copy
-        fm_2 = content.Format.objects.using(self.the_channel_id).get(format_size=51)
-        file_2 = content.File.objects.using(self.the_channel_id).get(format=fm_2)
-        api.update_content_copy(file_2, fpath_2)
+        fm_2 = content.FileFormat.objects.using(self.the_channel_id).get(extension='png')
+        file_2 = content.File.objects.using(self.the_channel_id).get(file_format=fm_2)
+        api.update_content_copy(channel_id=self.the_channel_id, file_object=file_2, content_copy_path=fpath_2)
         self.assertTrue(file_2.content_copy)
-        api.update_content_copy(file_2, None)
+        api.update_content_copy(channel_id=self.the_channel_id, file_object=file_2, content_copy_path=None)
         self.assertFalse(file_2.content_copy)
         content_copy_path = settings.CONTENT_COPY_DIR+'/3/3/335782204c8215e0061516c6b3b80271.mp4'
         self.assertTrue(os.path.isfile(content_copy_path))
 
         # all reference pointing to this content copy is gone,
         # the content copy should be deleted
-        api.update_content_copy(file_3, None)
+        api.update_content_copy(channel_id=self.the_channel_id, file_object=file_1, content_copy_path=None)
         self.assertFalse(os.path.isfile(content_copy_path))
         self.assertFalse(file_2.content_copy)
         self.assertFalse(file_2.checksum)
 
         # update None content copy on empty File object should be silent and have no effect
-        api.update_content_copy(file_2, None)
+        api.update_content_copy(channel_id=self.the_channel_id, file_object=file_2, content_copy_path=None)
 
         # test File __str__ method
-        self.assertEqual(file_1.__str__(), '09293abba61d4fcfa4e3bd804bcaba43.pdf')
+        self.assertEqual(file_3.__str__(), '09293abba61d4fcfa4e3bd804bcaba43.pdf')
 
         # test MimeType __str__ method
-        self.assertEqual(fm_1.mimetype.__str__(), 'video_high')
+        self.assertEqual(file_3.preset.name, 'high_res_video')
```

Comment:
Preset id's (like `high_res_video`) should be constants as well. And here it's saying that's the name -- shouldn't it be the ID? The name would be something human readable like "High Resolution".